docs/the-new-architecture/pillars-fabric-components.md

```diff
@@ -770,7 +775,64 @@ This command will look for all the dependencies of the project and it will insta
 
 #### Android
 
-#### JS
+Android configuration requires slightly more steps in order to be able to use our new Component.
+
+First, we need to enable the **New Architecture**, because **Fabric** requires it to run properly. This can be done by:
+
+1. Open the `android/gradle.properties` file
+2. Scroll down to the end of the file and switch the `newArchEnabled` property from `false` to `true`.
+
+Then, we need to instruct the `Android.mk` file that it needs to build also the new library.
+This can be with these steps:
+
+1. Open the `android/app/src/main/jni/Android.mk` file
+1. Add this line to include the library at the beginning of the file:
+
+   ```diff
+   include $(REACT_ANDROID_DIR)/Android-prebuilt.mk
+
+   # If you wish to add a custom TurboModule or Fabric component in your app you
+   # will have to include the following autogenerated makefile.
+   # include $(GENERATED_SRC_DIR)/codegen/jni/Android.mk
+
+   +include $(NODE_MODULES_DIR)/rnt-centered-text/android/build/generated/source/codegen/jni/Android.mk
+   include $(CLEAR_VARS)
+   ```
+
+1. In the same file, scroll down until you find a list of `libreact` libraries. There, we have to add the the library that has been generated. To do so, we need to add this line:
+   ```diff
+   libreact_codegen_rncore \
+   +react_codegen_RTNCenteredText \
+   libreact_debug \
+   ```
+
+:::note
+The exact name of the library can be found in the generated `Android.mk` that is located in the folder:
+`node_modules/rnt-centered-text/android/build/generated/source/codegen/jni/Android.mk`.
+The name of the library is the value of the `LOCAL_MODULE` variable of that file.
```

Comment:
I would not ask users to look into the generated Makefile. I think we should rephrase this to either:
- Mention that the library will be `libreact_codegen_<libraryname>` where `libraryname` is the one specified in the config
- Mention that this is not needed as libs will be autolinked